spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -150,7 +154,9 @@ private static void resolveTypeQualifierNicknames(AnnotationValue value, LinkedL
                         || annotationClass.equals(androidxNonNull)
                         || annotationClass.equals(eclipseNonNull)
                         || annotationClass.equals(eclipseNonNullByDefault)
-                        || annotationClass.equals(intellijNotNull)) {
+                        || annotationClass.equals(intellijNotNull)
```

Comment:
This is required to recognize `@NonNull` annotation. For some reason we had support for `@Nullable`, but not for `@NonNull`